sqlglot/expressions.py

```diff
@@ -1286,6 +1287,8 @@ def group_by(self, *expressions, append=True, dialect=None, copy=True, **opts):
         Returns:
             Select: the modified expression.
         """
+        if not expressions:
```

Comment:
how come?

Reply:
So the way that the PySpark DataFrame API thinks about this is that an empty group by prior to an aggregate is valid and represents grouping by nothing. This change allows the sqlglot builder to behave in the same way. If you would prefer for the builder to not behave that way then I can remove this and then do a check in the DataFrame API version.